common/src/main/java/io/netty/util/internal/InternalThreadLocalMap.java

```diff
@@ -43,6 +44,7 @@ public final class InternalThreadLocalMap extends UnpaddedInternalThreadLocalMap
     private static final ThreadLocal<InternalThreadLocalMap> slowThreadLocalMap =
             new ThreadLocal<InternalThreadLocalMap>();
     private static final AtomicInteger nextIndex = new AtomicInteger();
+    public static final int VARIABLES_TO_REMOVE_INDEX = nextVariableIndex();
```

Comment:
```suggestion
    static final int VARIABLES_TO_REMOVE_INDEX = nextVariableIndex();
```

Reply:
@normanmaurer  `VARIABLES_TO_REMOVE_INDEX ` is also used in `FastThreadLocal`, so the `public` modifier can not be removed.